src/EventStore.Client/EventStoreClientSettings.ConnectionString.cs

```diff
@@ -204,21 +220,45 @@ private static class ConnectionStringParser {
 					settings.ConnectivitySettings.TlsVerifyCert = (bool)tlsVerifyCert;
 				}
 
+				if (typedOptions.TryGetValue(TlsCaFile, out var tlsCaFile)) {
+					var tlsCaFilePath = Path.GetFullPath((string)tlsCaFile);
+					if (!string.IsNullOrEmpty(tlsCaFilePath) && !File.Exists(tlsCaFilePath)) {
+						throw new FileNotFoundException($"Failed to load certificate. File was not found.");
```

Comment:
As per my previous comment, we can instead use our new custom exception type, like so:
```csharp
throw new InvalidClientCertificateException("File not found");
```